core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -2650,6 +2650,7 @@ public void testRollbackDataOnStaleTransaction() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Why ignore?

Reply:
I knew you'd ask. This test does not do what it advertises. It is corrupt. It is meant to test bind variable interaction but insteat it sends wrong username/password and never gets to test anything. Instead it hangs occasionally.